sql/search_derived/search_metric_contribution/query.sql

```diff
@@ -0,0 +1,213 @@
+-- temp function returns how much of one metric is contributed by the quantile of another metric
+CREATE TEMP FUNCTION get_quantile(metric1 FLOAT64, metric2 FLOAT64, quantile FLOAT64) AS (
```

Comment:
This should be moved into the udfs directory, and given a more descriptive name. The name of the file you create there will be `udf/{udf_name}.sql`, and inside that file use `CREATE OR REPLACE FUNCTION udf.{udf_name}`.

Reply:
I added udf.quantile_search_metric_contribution and called this function in query. But the log showed a 'function not found' error. Why was this issue?

Reply:
Would you mind reviewing the new udf I added? Otherwise I can't add derived datasets I believe. Let me know if there were any problems! Thanks! :)